MIAUI/MIAUI/Views/LoginPage.xaml.cs

```diff
@@ -0,0 +1,10 @@
+namespace MIAUI.Views;
+
+public partial class LoginPage : ContentPage
+{
+	public LoginPage (LoginViewModel vm)
```

Comment:
Let's avoid passing the LoginViewModel as a parameter and instantiate it within the constructor instead, as has already been done with the other pages